packages/product-components/src/components/NetworkIcon/NetworkIcon.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { Meta, StoryObj } from '@storybook/react';
+import styled from 'styled-components';
+
+import { StoryColumn } from '@trezor/components';
+
+import { NetworkIcon } from './NetworkIcon';
+import { NETWORK_ICONS, isNetworkSymbolWithIcon } from '../../constants/networks';
+
+const Heading = styled.h2`
+    margin-bottom: 2px;
+`;
+
+const SubHeading = styled.p``;
+
+const NetworkName = styled.div`
```

Comment:
I suggest to use `<Paragraph margin={{bottom: 8}} variant="tertiary">...</Paragraph>` instead of this component